src/test/locale/fr.js

```diff
@@ -25,18 +25,19 @@ test('format', function (assert) {
     var a = [
             ['dddd, MMMM Do YYYY, h:mm:ss a',      'dimanche, février 14 2010, 3:25:50 pm'],
             ['ddd, hA',                            'dim., 3PM'],
-            ['M Mo MM MMMM MMM',                   '2 2 02 février févr.'],
+            ['M Mo MM MMMM MMM',                   '2 2e 02 février févr.'],
             ['YYYY YY',                            '2010 10'],
             ['D Do DD',                            '14 14 14'],
-            ['d do dddd ddd dd',                   '0 0 dimanche dim. Di'],
-            ['DDD DDDo DDDD',                      '45 45 045'],
-            ['w wo ww',                            '6 6 06'],
+            ['d do dddd ddd dd',                   '0 0e dimanche dim. Di'],
```

Comment:
You pronounce it "zéroième" ou "zérotième", thus the abbreviation with "e" is correct.

Reference:
Abréviations des adjectifs numéraux
http://www.academie-francaise.fr/abreviations-des-adjectifs-numeraux
from **Académie française**, the authority of French language rules.

This token `do` is actually pretty useless. When writing prose, in practice you will never use the `do` token. You will rather say with a cardinal "jour de la semaine 0" or "jour de la semaine numéro 0", and use the `d` token.